tx-source/site_core.php

```diff
@@ -1930,3 +1930,15 @@
 // Quotes/Formats
 $Definition['Toggle Quote'] = 'Toggle Quote';
 $Definition['Spoiler Warning'] = 'Spoiler Warning';
+
+// Catalogue Display Plugin/Formats
+$Definition['Default placeholder'] = 'Default placeholder';
```

Comment:
You don't need to duplicate the strings. Just pick the one file that is appropriate and put them there.

Reply:
oh ok good to know